packages/desktop/hooks/LocalCombatsContext/index.tsx

```diff
@@ -159,13 +173,42 @@ export const LocalCombatsContextProvider = (props: IProps) => {
 
       if (window.wowarenalogs.logs?.handleActivityStarted) {
         window.wowarenalogs.logs.handleActivityStarted((_nodeEvent, activityStartedEvent) => {
-          // eslint-disable-next-line no-console
-          console.log(activityStartedEvent);
+          if (!currentActivity) {
+            currentActivity = activityStartedEvent;
+            window.wowarenalogs.obs?.startRecording && window.wowarenalogs.obs.startRecording();
```

Comment:
not sure which TS version it is but I believe there's a syntax sugar that allows you to do `window.wowarenalogs.obs?.startRecording.()`

Reply:
`startRecording?.()` TIL 👍